features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/GrokParserStageSequenceBuilder.java

```diff
@@ -512,6 +526,10 @@
 						factory.stringUntilWhitespace(semanticStringToField(semanticString));
 						factory.whitespace();
 						break;
+					case WHITESPACE:
+						LOG.warn("WHITESPACE pattern followed immediately by another pattern will greedily consume until non-whitespace is encountered");
```

Comment:
Is this log necessary? Seems like something that should be in the documentation rather than being logged at run-time.

Since this pattern is actually used in the default configuration these warnings will always be printed at startup when using the RadixTreeParser.